mailbox/cassandra/pom.xml

```diff
@@ -260,6 +255,12 @@
                     <artifactId>throwing-lambdas</artifactId>
                     <version>0.5.0</version>
                 </dependency>
+                <dependency>
```

Comment:
http://projects.lidalia.org.uk/slf4j-test/.
I think this is nice, you do not like it?